### DISCUSSION TITLE
Spacer and Separator Widgets For Configure Screen

### DISCUSSION THREAD

**Discussion by Sonwon1** [osmandapp/OsmAnd#17915] [Ideas]
For Configure Screen, 

I would like a spacer widget with these options, blank and a line. It would also be nice if they could  be selectable for full widget or half a widget in height.

And a separator widget that would force the widgets away from each other so there would be one group of widgets at one end and another group of widgets at the other end of the panel. I would envision these only being useful on the left and right panels. Maybe even allow up to two separator widgets allowing three groupings of widgets.

**Comment by vshcherb** [comment]
Nice suggestion
